### DISCUSSION TITLE
Küre üzerindeki noktalar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#110] [Zeka Sorusu]
Hiçbir noktanın başka hiçbir noktaya 1 birimden daha yakın olmaması koşuluyla, bir birim kürenin yüzeyine en fazla kaç nokta yerleştirilebilir?
Not: -Birim kürenin yarıçapı 1 birimdir.

**Comment by Ersan0102** [comment]
Bu birimküre 3 tane axis'e ayrılırsa her axiste 4 tane nokta olur **(kesişen noktalar sayılmayacak elbette)**. r = 1 olduğu için axis'in çevresi 6 birim olur. fakat 6 birimin her birine bir nokta koyarsak noktaların arasındaki mesafe 1 birimden kısa olur. Bu yüzden axislerdeki noktalar kesişen yerleri çıkarıldığında toplam **6** nokta içerebilir.
